plugins/woocommerce/tests/e2e-pw/tests/merchant/customer-list.spec.js

```diff
@@ -85,7 +85,7 @@ test.describe( 'Merchant > Customer List', { tag: '@services' }, () => {
 
 	test(
 		'Merchant can view a list of all customers, filter and download',
-		{ tag: '@skip-on-default-pressable' },
+		{ tag: [ '@skip-on-default-pressable', '@skip-on-default-wpcom' ] },
```

Comment:
Since we're using the same site every time, we're not getting expected numbers here.